run-dask-process.sh

```diff
@@ -128,13 +123,12 @@ function buildUCXWithInfinibandArgs {
                 --scheduler-file $SCHEDULER_FILE
                "
 
-    WORKER_ARGS="--enable-tcp-over-ucx
-                --enable-nvlink
-                --enable-infiniband
-                --enable-rdmacm
+    WORKER_ARGS="--interface=$DASK_CUDA_INTERFACE
```

Comment:
Just wondering why you removed some of the arguments passed to the workers and added the interface $DASK_CUDA_INTERFACE. Does it embody those arguments(--enable-nvlink, --enable-infiniband, ...)?

Reply:
Is it because we ared oing something similar on wait_for_workers.py?